packages/graphql-crud/src/validateUpdateInputData.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  getNullableType,
+  GraphQLObjectType,
+  GraphQLSchema,
+} from 'graphql';
+
+import {
+  isEmpty,
+  isPlainObject,
+} from 'lodash';
+
+export interface ValidateUpdateInputDataProps {
+  type: GraphQLObjectType;
+  schema: GraphQLSchema;
+  data: object;
+}
+
+// For every null value in the input data
+// check that it can be nullable by check the type definition.
+export const validateUpdateInputData = (props: ValidateUpdateInputDataProps) => {
+  if (isEmpty(props.data)) {
+    throw new Error('data input object is missing');
+  }
+
+  const isNonNullable = (type) => type.astNode.type.kind === 'NonNullType';
```

Comment:
Why is this function nested? It could be defined at the outermost scope.

Reply:
Agreed. I have it moved to the outermost scope  in an upcoming PR.